eo-maven-plugin/src/main/java/org/eolang/maven/OyRemote.java

```diff
@@ -34,34 +38,41 @@
  *
  * @since 0.1
  */
+@SuppressWarnings({"PMD.AvoidDuplicateLiterals",
+    "PMD.ConstructorOnlyInitializesOrCallOtherConstructors"})
 public final class OyRemote implements Objectionary {
 
+    /**
+     * Tag.
+     */
+    private final String tag;
+
     /**
      * The address template.
      */
-    private final String template;
+    private String template;
```

Comment:
@yasamprom we try to keep our objects immutable: https://www.yegor256.com/2014/06/09/objects-should-be-immutable.html